openpype/hosts/flame/plugins/load/load_clip.py

```diff
@@ -38,8 +47,16 @@ def load(self, context, name, namespace, options):
         version_name = version.get("name", None)
         colorspace = self.get_colorspace(context)
 
+        # in case output is not in context replace key to representation
+        if not context["representation"]["context"].get("output"):
+            self.clip_name_template = self.clip_name_template.replace(
```

Comment:
I just realized that if the load plugin is used for loading of 2+ representations and only one of them does not have `"output"` the template for others will be broken. My recommendation is to not change value on `self`.